### DISCUSSION TITLE
How to make Stash acceccible from Web (QNAP)

### DISCUSSION THREAD

**Discussion by Asuryan34** [stashapp/stash#4272] [Q&A]
Hello.
I'm trying to make Stash accessibly from the Web and .. I'm just lost
First: I could do it on my old synology NAS without much problem
Second: Yes, I added a user ID/password and remove the line I was suposed to remove in the config file to allow the access from the web

My problem, I think is more from QNAP/ Contenair station / Docker

I followed these steps:  https://www.qnap.com/en/how-to/tutorial/article/how-to-configure-the-default-web-url-port-for-containers-and-applications-in-container-station-3

But, I don't gat any "Web URL" and my port forwarding is stuck on 9999 --> 9999TCP even if I changed the port.

Any idea ?

**Comment by ofjsur** [comment]
Can you just forward the localhost port that stash runs on to your local machine?

**Comment by ALonelyJuicebox** [comment]
I really _really_ do not advise exposing Stash to the open web. You are literally asking for problems.
Instead, consider accessing Stash via a VPN connection that gives you access to your local network.

**Comment by Asuryan34** [comment]
Hello.

Yes, I just realized that when my ubiquity system said "we do not recommend
opening a port".

They suggested using a VPN like "WireGuard". I'll do that.

Thank you for your responses. :)
